pepper-apis/src/main/java/org/broadinstitute/ddp/json/Profile.java

```diff
@@ -23,26 +18,36 @@
     public static final String PREFERRED_LANGUAGE = "preferredLanguage";
     public static final String FIRST_NAME = "firstName";
     public static final String LAST_NAME = "lastName";
+
     @SerializedName(BIRTH_MONTH)
     private Integer birthMonth;
     @SerializedName(BIRTH_YEAR)
     private Integer birthYear;
     @SerializedName(BIRTH_DAY_IN_MONTH)
     private Integer birthDayInMonth;
     @SerializedName(SEX)
-    private Sex sex;
```

Comment:
wonder if reason for using a enum is to have more control over this and now we can pass any String value.
Is that ok ?

Reply:
ok. I see Enum is moved to UserProfile.java (UserProfile.SexType) . 
should be fine.